compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -8532,7 +8680,10 @@ TR::Node *dmulSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
       // In IEEE FP Arithmetic, d * 1.0 is d
       // Pretend to be an int before doing the comparison
       //
-      BINARY_IDENTITY_OP(LongInt, DOUBLE_ONE)
+      auto binOpSimplifier = getLongBinaryOpSimplifier(s);
+      auto identity = binOpSimplifier.tryAndSimplifyIdentityOp(node, (int64_t) DOUBLE_ONE);
```

Comment:
I think to be consistent with the rest of the change from this PR on double where `getDoubleBitsBinaryOpSimplifier` is used if the children are double (`dmulSimplifier`), this one should be updated to `getDoubleBitsBinaryOpSimplifier` instead of using `getLongBinaryOpSimplifier`. 

Again since this changes the original code instead of just refactoring, as mentioned earlier, some regression test should be done.